### ISSUE TITLE
Sequential model must get an `input_shape`

### ISSUE THREAD

**Issue by hengfun** [farizrahman4u/seq2seq#96]
from keras.preprocessing import sequence
from keras.models import Sequential
from keras.layers import Dense, Dropout, Embedding, LSTM, Input, Merge
from keras.layers.core import Dense, Activation  
# from keras.datasets import imdb
from keras.layers import recurrent
from keras.layers.recurrent import Bidirectional

model = Sequential()
model.add(Bidirectional(LSTM(10, input_dim=(10, 20))))

Exception: The first layer in a Sequential model must get an `input_shape` or `batch_input_shape` argument.

not sure why the example from the code base isnt working, any ideas?

**Comment by farizrahman4u** [comment]
model.add(Bidirectional(LSTM(10), input_shape=(10, 20)))

**Comment by AndreiBarsan** [comment]
Just a small question (perhaps I am misunderstanding the architecture), but wouldn't it be possible for the `Bidirectional` wrapper to automatically expose the `input_shape` of the cell it wraps, if available?
